Bottom: 1944ff9455330a5668da3d69575cd2e1928f07dc
Top:    e0b82367be775902f3588a79dd714486d21646b3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-19 15:14:58 -0700

Refresh of rstream-fork

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 490caa5..befb7c6 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -36,6 +36,7 @@
 #include <sys/types.h>
 #include <sys/socket.h>
 #include <sys/time.h>
+#include <sys/wait.h>
 #include <netdb.h>
 #include <fcntl.h>
 #include <unistd.h>
